YubiKit/YubiKit/Connections/NFCConnection/Sessions/YKFNFCOTPSession.m

```diff
@@ -74,29 +74,29 @@ - (void)start {
 - (void)readerSession:(NFCNDEFReaderSession *)session didInvalidateWithError:(NSError *)error {
     YKFAssertOnMainThread();
     
-    self.nfcSession = nil;
     if ([self shouldIgnoreError: error]) {
         return;
     }
     
     YKFLogNSError(error);
     
-    self.nfcOTPResponseBlock(nil, error);
+    if (self.nfcOTPResponseBlock) self.nfcOTPResponseBlock(nil, error);
```

Comment:
Should we prefer to always use curly braces for if-statements? Is there a "best practice" for this for Obj-C?